### DISCUSSION TITLE
Question about running testng tests without breakpoint

### DISCUSSION THREAD

**Discussion by krish-r** [mfussenegger/nvim-jdtls#316] [Q&A] [answered]
Hello,

I'm able to successfully call the functions `require'jdtls'.test_class()` or `require'jdtls'.test_nearest_method()` inside -
- `junit` test class/methods with and without breakpoints
- `testng` test class/methods with breakpoints

However, if I try to call the functions inside a `testng` test class/methods without any breakpoints, the tests are not detected.
(I'm getting the following message - "**Tests finished. Results printed to dap-repl. All 0 succeeded**").

Is this the expected behavior? Or is there an option to run the functions inside testng test class/methods without setting breakpoints?

Thanks in advance.

```java
// TestNGExampleTest.java
// A sample test class

package com.example;

import org.testng.Assert;
import org.testng.annotations.Test;

public class TestNGExampleTest {

  @Test
  public void testMethodTest() {
    int x = 10;
    int y = 10;
    Assert.assertEquals(x + y, 200);
  }
}

```

**Comment by mfussenegger** [comment, marked as answer]
I think the tests are run and there should be output in the repl or integrated terminal, but the results are not parsed. The result parser is currently only implemented for junit.

**Reply by krish-r** [reply]
Ah! my bad. Yes, you are correct. 
I can see the test results in the dap-terminal. Looks like I had commented the console option in my dap-ui config.
Thanks.
